test/EFCore.PG.FunctionalTests/Query/ArrayListQueryTest.cs

```diff
@@ -312,6 +312,18 @@ public void All_Contains()
 
         #endregion
 
+        #region ArrayToString
+        [Fact]
+        public void String_Join_List()
```

Comment:
Can you please replicate the new tests that you added in ArrayArrayQueryTest to here? The idea is to maintain identical functionality between the two, and having the same tests helps.